third_party/amd/lib/TritonAMDGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -107,6 +108,13 @@ struct ConvertTritonAMDGPUToLLVM
     ModuleMembarAnalysis membarPass(&allocation);
     membarPass.run();
 
+    // Collect all the assumed expressions
+    DenseSet<Value> assumptions;
+    mod.walk([&](Operation *op) {
+      if (op->getName().getStringRef() == "llvm.intr.assume")
```

Comment:
We already include the LLVM dialect so you should be fine to check the op with `isa`?